doc/source/theory/Mellin.rst

```diff
@@ -56,7 +56,7 @@ Plus Distributions
 ^^^^^^^^^^^^^^^^^^
 
 .. math ::
-    \mathcal M[1/(1-x)_+](N) = S_1(N)
+    \mathcal M[1/(1-x)_+](N) = - S_1(N - 1)
```

Comment:
The minus sign is fine. I was not sure about the `N - 1`, but after checking I realized that is coming from the `N - 1` in the Mellin transform (`N - 2` after division, again `N - 1` after integration).

It should be correct, well-spotted!

Reply:
This is a typo indeed